Helper/OmiseHelper.php

```diff
@@ -85,6 +85,100 @@ class OmiseHelper extends AbstractHelper
         CcGooglePay::CODE
     ];
 
+    /**
+     * @var array
+     */
+    private $omisePaymentMethods;
+
+    /**
+     *
+     * @var array
+     */
+    private $omiseCodeByOmiseId = [
+        // card payment
+        Config::ID => Config::CODE,
+        CcGooglePay::ID => CcGooglePay::CODE,
+
+        // offsite payment
+        Alipay::ID => Alipay::CODE,
+        Truemoney::ID => Truemoney::CODE,
+        Pointsciti::ID => Pointsciti::CODE,
+        Fpx::ID => Fpx::CODE,
+        Alipayplus::ALIPAY_ID => Alipayplus::ALIPAY_CODE,
+        Alipayplus::ALIPAYHK_ID => Alipayplus::ALIPAYHK_CODE,
+        Alipayplus::DANA_ID => Alipayplus::DANA_CODE,
+        Alipayplus::GCASH_ID => Alipayplus::GCASH_CODE,
+        Alipayplus::KAKAOPAY_ID => Alipayplus::KAKAOPAY_CODE,
+        Alipayplus::TOUCHNGO_ID => Alipayplus::TOUCHNGO_CODE,
+        Rabbitlinepay::ID => Rabbitlinepay::CODE,
+        Ocbcpao::ID => Ocbcpao::CODE,
+        Grabpay::ID => Grabpay::CODE,
+
+        // offsite internet banking payment
+        Internetbanking::BBl_ID => Internetbanking::CODE,
```

Comment:
```suggestion
        Internetbanking::BBL_ID => Internetbanking::CODE,
```
Should it be big L? 🤔